controllers.js

```diff
@@ -21,8 +19,12 @@ exports.getTopics = (request, response, next) => {
 };
 
 exports.getArticles = (request, response, next) => {
-  fetchArticles()
+  const { topic } = request.query;
```

Comment:
you can destructure these 3 at the same time `{ blah1, blah2, blah3 } = request.query`